### PR TITLE
C++11 threading

### PR DESCRIPTION
**Pull request by scrawl**
Just asking travis for a test run.

### CODE REVIEW

**Comment by psi29a** [comment]
computer says no

**Comment by scrawl** [comment]
Ready for merge.

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ language: cpp
 branches:
   only:
     - master
+    - c++11
```
Comment (psi29a) [inline .travis.yml]: Was this supposed to stay in this PR?

Comment (scrawl) [inline .travis.yml]: I couldn't be bothered to undo it. Is there any harm leaving it in?

Comment (psi29a) [inline .travis.yml]: Not really, I'll do it when you're not looking though. :P

**Pull request merged**